e2e/config.py

```diff
@@ -0,0 +1,78 @@
+import pytest
```

Comment:
some comments over file on what the intended action will help future reader to understand the code.

Reply:
Agreed, add [docstrings](https://www.python.org/dev/peps/pep-0257/#what-is-a-docstring) to functions specially those in the utility files such as `cluster.py` and `config.py`.